core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/ApacheSender43.java

```diff
@@ -71,8 +73,14 @@ private ApacheSender43() {
     }
 
     @Override
-    public HttpResponse sendPostRequest(HttpPost post) throws IOException {
-        return getHttpClient().execute(post);
+    public HttpResponse sendPostRequest(HttpPost post) throws IOException, FriendlyException {
+        try {
+            return getHttpClient().execute(post);
+        } catch (SSLHandshakeException e) {
+            throw new FriendlyException("ApplicationInsights Java Agent failed to send telemetry data.",
+                    "Unable to find valid certification path to requested target",
+                    "Please import the Breeze certificate into the java key store.");
```

Comment:
can we log the hostname here, so they know which host they need to import the SSL cert for?